.github/workflows/ci-nix.yml

```diff
@@ -20,6 +20,7 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   lint:
+    if: github.repository == 'fedimint/fedimint'
```

Comment:
I think this should work on forks if we only set the `authToken` if it's available and gives at least rudimentary feedback

Reply:
What doesn't work about lints on forks? cachix uses `continue-on-error: true`

Reply:
Verified `continue-on-error: true` will work on forks 👍 

This is a successful run on my fork's master branch: https://github.com/bradleystachurski/fedimint/actions/runs/6128318965